discovery-provider/plugins/pedalboard/src/models.ts

```diff
@@ -0,0 +1,959 @@
+// The TypeScript definitions below are automatically generated.

```

Comment:
I feel like we've done this a few times now, maybe we should try to standardize this approach -- or maybe this is the standard and we should discuss how something like the es-indexer evolves from this. fyi @stereosteve

Reply:
yeah I agree, maybe it makes sense as its own module since it's so widely used?